packages/queries/src/hashQuery.ts

```diff
@@ -0,0 +1,55 @@
+import { AnyQuery } from './types/AnyQuery';
+
+// Copied from: https://github.com/TanStack/query/blob/353e4ad7291645f27de6585e9897b45e46c666fb/packages/query-core/src/utils.ts#L205
+/**
+ * Default query & mutation keys hash function.
+ * Hashes the value into a stable hash.
+ */
+export function hashQuery(query: AnyQuery): string {
+    return JSON.stringify(query, (_, val) =>
```

Comment:
Be aware that you will need to modify the hash when you introduce query parameters.

`hashQuery(from().where({id:param('id',1)}).many())`

should give the same result as

`hashQuery(from().where({id:param('id',2)}).many())`

Reply:
So probably we should do that already in this PR